### PR TITLE
Stress testing

### PR DESCRIPTION
**Pull request by yerx**
- added New Relic
- added K6 script for load testing

### CODE REVIEW

File: database/postgres/index.js [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+//       ));
+//       return filteredSellerItems;
+//     })
+//     .catch((err) => {
+//       console.log(err.stack);
+//     })
+//     .then(() => client.end());
+// };
+
 const pool = new Pool({
```
Comment (dmoygit) [inline database/postgres/index.js:35]: Should we use Pool now or should we use it as part of our optimization?

File: server/stress-test/k6script.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+// export default function () {
+//   http.get('http://localhost:3004/?36256214');
+//   http.get('http://localhost:3004/additional/36256214');
+// }
+
+
+import http from 'k6/http';
+import { check } from 'k6';
+import { Rate, Counter, Trend } from 'k6/metrics';
```
Comment (dmoygit) [inline server/stress-test/k6script.js:16]: Very nice!  You used added metrics too!

**Review by dmoygit — APPROVED** [review]
lgtm!

File: database/postgres/index.js (hunk 1 above)
Comment (yerx) [reply database/postgres/index.js:35]: I'm not sure. I was having trouble with Client so I used Pool.

**Pull request merged**